### DISCUSSION TITLE
Stream from youtube and other platforms

### DISCUSSION THREAD

**Discussion by ghost** [mediacms-io/mediacms#72] [Q&A]
Is it possible to stream from a source?
For example If I manage to get a stream URL

**Comment by mgogoulos** [comment]
It's not something that we were planning to work in the near future, however this functionality is worth to be explored

**Reply by gam-ragnar** [reply]
@grubeli IMHO streaming off a URL should be a bounty feature. Core dev got more generic issues to implement still.
